packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -177,7 +245,13 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
             </div>
 
             <div
-                data-ng-if="$ctrl.choosedAdditionalOptions.length === 0 || (!$ctrl.process.selectedOffer.options.gtr_ovh && !$ctrl.process.selectedOffer.options.gtr_ovh.selected)"
+                data-ng-if="$ctrl.choosedAdditionalOptions.length === 0 ||
+                            ((!$ctrl.process.selectedOffer.options.gtr_10ho && !$ctrl.process.selectedOffer.options.gtr_10ho.selected) &&
+                            (!$ctrl.process.selectedOffer.options.gtr_4ho && !$ctrl.process.selectedOffer.options.gtr_4ho.selected) &&
+                            (!$ctrl.process.selectedOffer.options.gtr_5m_10ho && !$ctrl.process.selectedOffer.options.gtr_5m_10ho.selected) &&
+                            (!$ctrl.process.selectedOffer.options.gtr_5m_4ho && !$ctrl.process.selectedOffer.options.gtr_5m_4ho.selected) &&
+                            (!$ctrl.process.selectedOffer.options.gtr_20m_10ho && !$ctrl.process.selectedOffer.options.gtr_20m_10ho.selected) &&
+                            (!$ctrl.process.selectedOffer.options.gtr_20m_4ho && !$ctrl.process.selectedOffer.options.gtr_20m_4ho.selected))"
```

Comment:
This logic should be stored in a method. You could check for all objects in `option`, with some to check if one option is selected

Reply:
done thx